python/lsst/ci/prepare.py

```diff
@@ -14,14 +18,19 @@
 import re
 import pipes
 import subprocess
-import collections
 import abc
 import yaml
 import copy
 
-from . import tsort
+from .eups import EupsModule
 
 from .git import Git, GitError
+from .models import Product, RepoSpec, ProductIndex, Ref, DEFAULT_BRANCH_NAME
```

Comment:
I strongly recommend `from . import models` instead (and referencing the names with `module.` in front of them) for clarity. The names have no obvious connection to the module name, so when reading the code it is not obvious where these names come from.